SalesforceCommerceCloud/pwa-kit
# [Retail App] Order summary prices on check out page misaligned on mobile

# Description
Fix for the issue: https://github.com/SalesforceCommerceCloud/pwa-kit/issues/198
**Ready for Review**

# Types of Changes




> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# Changes

- Replaced the stack container with a box container
- Added flex="1" for the box container

# How to Test-Drive This PR

1. npm start
2. Add an item to cart.
3. Go to Cart > Checkout page.
4. Scroll down to Order Summary.
5. Click the arrow to see the item in cart.

# Working Examples

**Mobile View**
<img width="1792" alt="Screenshot 2021-11-26 at 2 35 51 AM" src="https://user-images.githubusercontent.com/83722798/143499883-184101c0-c0e7-4024-b9d9-addd009ab028.png">

**Tablet View**
<img width="1792" alt="Screenshot 2021-11-26 at 2 37 00 AM" src="https://user-images.githubusercontent.com/83722798/143499915-b870bbb1-ba39-4909-9b17-0c1a878baa51.png">

**Desktop View**
<img width="1792" alt="Screenshot 2021-11-26 at 2 37 29 AM" src="https://user-images.githubusercontent.com/83722798/143499935-d3ca248e-2308-4adb-9362-381772c8a182.png">

# Checklists





## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
